cpp/oneapi/dal/algo/pca/test/batch.cpp

```diff
@@ -211,6 +214,28 @@ TEMPLATE_LIST_TEST_M(pca_batch_test, "pca common flow", "[pca][integration][batc
     this->general_checks(data, component_count, data_table_id);
 }
 
+TEMPLATE_LIST_TEST_M(pca_batch_test,
+                     "pca common flow normal distribution",
+                     "[pca][integration][batch]",
+                     pca_types) {
+    SKIP_IF(this->not_available_on_device());
+
+    const te::dataframe data =
+        GENERATE_DATAFRAME(te::dataframe_builder{ 100, 10 }.fill_normal(5.0, 2.0),
+                           te::dataframe_builder{ 100000, 10 }.fill_normal(5.0, 1.5));
+
+    // Homogen floating point type is the same as algorithm's floating point type
+    const auto data_table_id = this->get_homogen_table_id();
+
+    const std::int64_t component_count = GENERATE_COPY(0,
+                                                       1,
+                                                       data.get_column_count(),
+                                                       data.get_column_count() - 1,
+                                                       data.get_column_count() / 2);
+
+    this->general_checks(data, component_count, data_table_id);
+}
```

Comment:
I don't think we need normal distribution tests for PCA. Please consult with @vlad-nazarov, he needed normal distribution for kernel functions testing